test/functional/test_framework/test_node.py

```
@@ -785,11 +784,14 @@ def check_peers():
                     if p['subver'] == p2p.strSubVer:
                         return False
             return True
-        wait_until_helper(check_peers, timeout=5)
+        self.wait_until(check_peers, timeout=5)
 
         del self.p2ps[:]
 
-        wait_until_helper(lambda: self.num_test_p2p_connections() == 0, timeout_factor=self.timeout_factor)
+        self.wait_until(lambda: self.num_test_p2p_connections() == 0)
+
+    def wait_until(self, test_function, timeout=60):
```

Ruff (0.12.2)

Missing return type annotation for public function `wait_until`

(ANN201)